packages/terra-compact-interactive-list/src/CompactInteractiveList.jsx

```diff
@@ -223,8 +223,9 @@ const CompactInteractiveList = (props) => {
   };
 
   const onFocus = (event) => {
-    if (!event.currentTarget.contains(event.relatedTarget)) {
+    if (!event.currentTarget.contains(event.relatedTarget) || event.target === listRef?.current) {
```

Comment:
Why do we need the OR condition.  Won't this cause the event to trigger every time focus is changed within the list?  If that is what you truly need, then there is no purpose for the first part of this condition.

Reply:
reverted adding that as I can't reproduce the bug it was meant to fix anymore